Makefile

```diff
@@ -3,6 +3,11 @@ format-check:
 	@echo "Checking format..."
 	python -m black src/
 
+.PHONY: format-isort
+format-isort:
+	@echo "Checking imports..."
+	python -m isort --check src/ --profile black
```

Comment:
does this use a non-zero exit code if the files were not formatted correctly (in addition to fixing the imports)? Just want to make sure that the CI job will fail if the imports are not sorted.